SECURITY.md

```diff
@@ -0,0 +1,32 @@
+# Security Policy
+
+## Supported Versions
+
+Only the latest release of CodiMD is supported. We don't have the
```

Comment:
I would even argue that even _with_ enough resources there would be little to no benefit in supporting legacy versions.